src/sass/patternfly.scss

```diff
@@ -1,4 +1,4 @@
-@import "~patternfly/dist/sass/patternfly/variables";
+@import "../../node_modules/patternfly/dist/sass/patternfly/variables";
```

Comment:
@nicpuddu , this seems to cause an error when I run `npm install` since `patternfly` belongs to `peerDependencies`. Whenever I run install, it will preemptively remove all existing dependencies in `npm_modules` before downloading dependencies, then executes `npm run prepublish` after, hence running your babel script. What I did in my workaround is to proceed and execute `npm-install-peers` then re-run `npm run prepublish` to continue where npm stopped before on installing. 

My thoughts is maybe this solves by moving/placing `patternfly` from `peerDependencies` to `devDependencies`? What do you think?

Reply:
sounds fine to me, I'll apply the change now